env-var-docs/generate_markdown.py

```diff
@@ -160,6 +160,13 @@ def append_node_to_out(node: env_var_docs.parser.Node):
         if group is not None:
             desc = group.group_description
         if var is not None:
+            if var.mode == env_var_docs.parser.Mode.ADMIN_WRITEABLE:
```

Comment:
suggestion (non-blocking): Please also update the readme [here](https://github.com/civiform/civiform/tree/main/env-var-docs/parser-package) to include a sentence about `ADMIN_WRITEABLE` under "mode".

Reply:
Good catch, thanks!

Reply:
Done